Oqtane.Client/Modules/ModuleBase.cs

```diff
@@ -10,11 +10,12 @@ namespace Oqtane.Modules
 {
     public class ModuleBase : ComponentBase, IModuleControl
     {
-        public Logger logger { get; set; }
+        private Logger _logger;
+
+        protected Logger logger => _logger ?? (_logger = new Logger(this));
```

Comment:
Lazy loading of logger, it was instantiated on every module loaded from database